### ISSUE TITLE
关于MHCN代码的Loss function问题请教。

### ISSUE THREAD

**Issue by shaitaiyangmie** [PaddlePaddle/PaddleRec#954]
请问在计算自监督loss的时候，局部互信息的部分为何是local_loss = paddle.sum(-paddle.log(F.sigmoid(pos - neg1)) -paddle.log(F.sigmoid(**neg1** - neg2)))？为什么不是pos-neg2而是neg1-neg2？
